styleguide/components/pagination/pagination-docs.mdx

```diff
@@ -23,3 +21,25 @@ Supplementary component used alongside pagination to show how many results have
 <Preview>
   <Story id="components-pagination--paging-info" />
 </Preview>
+
+## Using with Rails
+
+If you are using the `citizens_advice_components` gem you call the component from within a template using:
+
+```rb
+render(CitizensAdviceComponents::PaginationComponent.new(
+  current_params: request.query_parameters,
+  num_pages: 100,
+  current_page: 100
+))
+```
+
+The component assumes that you are using `page` as the query otherwise it will respect any other passed in from `current_params`.
```

Comment:
I realised whilst writing this that we are force the query parameter used for pagination. Maybe that should be configurable?

Reply:
Might be worth looking at https://github.com/kaminari/kaminari for best practice? If I recall correctly, it provides default names for these params but allows them to be overridden if people _really_ need it.

Reply:
I'm not sure I understand the English explanation above. Does `respect` mean "pass through"?

Reply:
That's correct, pass through is much clearer.